src/OSM_dataTool.js

```diff
@@ -25,7 +25,7 @@ module.exports = function ({ routes, ways, assumeFirstWayIsStart, mapProperties,
             debug(`${data.points.length} points in route`)
             data.points = filterPoints(data.points)
             debug(`${data.points.length} points after filtering`)
-
+            current_route.tags.id = current_route.id
```

Comment:
Let's not modify the original data, but rather merge the id in when we generate the GeoJSON Feature.

```suggestion
            geojson_features.push({
                "type": "Feature",
                "properties": mapProperties({ ...current_route.tags, id: current_route.id }),
                "geometry": {
                    "type": "LineString",
                    "coordinates": data.points,
                    "nodes": data.nodes,
                }
            })
```